headers/types/common/enums.h

```diff
@@ -3872,4 +3872,15 @@ enum debug_log_flag {
     DEBUG_LOG_FLAG_PERFORMANCE = 16,
 };
 
+// Fade status. Fade in means increasing brightness and fade out means decreasing.
+// As a result, fading the screen to white is considered a fade in.
+enum fade_status { FADE_NONE = 0, FADE_COMPLETE = 1, FADE_IN = 2, FADE_OUT = 3 };
```

Comment:
Enum definitions usually span multiple lines

Reply:
These ones I originally had on multiple lines but running `make format` made it like this. Not sure what to do about that.

Reply:
I think if you add a trailing comma after the final value `FADE_OUT`, it should keep the multiline format.

Reply:
Alright that should be fixed now.